### DISCUSSION TITLE
How does CapsNumLockIndicator widget fmt formatting work?

### DISCUSSION THREAD

**Discussion by tamis-laan** [qtile/qtile#2612] [Q&A]
I'm using the CapsNumLockIndicator widget inside my bar and would like to change the formatting. In order to do this I have to set the fmt variable, however it's not clear how this is done? What syntax should I use?

**Comment by ramnes** [comment]
What is the current output and what output do you want to achieve?

**Reply by tamis-laan** [reply]
Current output is `Caps off Num off` and I would simply like to show a `C`, `N` or `CN` and nothing `` when both are off.

**Reply by ramnes** [reply]
There is no way to do this at the moment; PR welcome!
